### PR TITLE
FHIR Persistence schema updateSchema has a second call to the updateProc

### PR DESCRIPTION
**Pull request by prb112**
which shouldn't be necessary. #1781

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/BaseObject.java [hunk 1]
```diff
@@ -256,7 +256,8 @@ public void applyTx(IDatabaseAdapter target, ITransactionProvider tp, IVersionHi
     @Override
     public void applyVersion(IDatabaseAdapter target, IVersionHistoryService vhs) {
         // Only for Procedures do we skip the Version History Service check, and apply.
-        if (vhs.applies(getSchemaName(), getObjectType().name(), getObjectName(), version) || getObjectType().equals(DatabaseObjectType.PROCEDURE)) {
+        if (vhs.applies(getSchemaName(), getObjectType().name(), getObjectName(), version)
```
Comment (tbieste) [inline fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/BaseObject.java:259]: Update copyright

File: fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/SchemaGeneratorUtil.java [hunk 2]
```diff
@@ -33,6 +34,7 @@ public static String readTemplate(String adminSchemaName, String schemaName, Str
         }
```
Comment (tbieste) [inline fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/SchemaGeneratorUtil.java:34]: Update copyright

**Comment by tbieste** [comment]
Left a couple minor comments.  Otherwise, LGTM.

**Comment by prb112** [comment]
> Left a couple minor comments. Otherwise, LGTM.

fixed

**Review by lmsurpre — APPROVED** [review]
LGTM

**Pull request merged**